src/components/static/prelive/about/About.jsx

```diff
@@ -1,5 +1,14 @@
 const About = () => {
-  return <div>About</div>;
+  return (
+    <div className="w-2/5 h-500 bg-cutie-blue-200/75 rounded-[73px] text-4xl font-bold p-5 leading-relaxed">
```

Comment:
try the biggest rounded tailwind has, it is find if it's a bit off from the design